sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -183,16 +184,19 @@ export abstract class MessageReceiver extends LinkEntity<Receiver> {
   /**
    * Creates a new AMQP receiver under a new AMQP session.
    */
-  protected async _init(options: ReceiverOptions, abortSignal?: AbortSignalLike): Promise<void> {
+  protected async _init(
+    options: ReceiverOptions,
+    operationOptions: OperationOptionsBase
+  ): Promise<void> {
     try {
-      await this.initLink(options, abortSignal);
+      await this.initLink(options, operationOptions);
 
       // It is possible for someone to close the receiver and then start it again.
       // Thus make sure that the receiver is present in the client cache.
       this._context.messageReceivers[this.name] = this as any;
     } catch (err) {
       err = translateServiceBusError(err);
-      logger.logError(err, "%s An error occured while creating the receiver", this.logPrefix);
+      logger.logError(err, "%s An error occurred while creating the receiver", this.logPrefix);
```

Comment:
Do we have approval for this API change?

Reply:
I consider it a bug fix. 😁

Reply:
@ramya-rao-a

Reply:
@xirzec @bterlson

Reply:
I'm pretty sure @richardpark-msft is joking :smile: